File: app/src/main/java/fr/istic/mob/network/networkadrk/models/Graph.java
package fr.istic.mob.network.networkadrk.models;

import java.util.List;

public class Graph {
    private List<Object> nodes;
    private List<Object> connections;

    public List<Object> getNodes() {
        return nodes;
    }

    public void setNodes(List<Object> nodes) {
        this.nodes = nodes;
    }

    public List<Object> getConnections() {
        return connections;
    }

    public void setConnections(List<Object> connections) {
        this.connections = connections;
    }
}

File: app/src/main/java/fr/istic/mob/network/networkadrk/models/Graph.kt
package fr.istic.mob.network.networkadrk.models;

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

data class Graph (
    val nodes: MutableList<Node> = mutableListOf(),
    val connections: MutableList<Connection> = mutableListOf()
) : Serializable
